src/elements/alt-style/AltStyleElementForm.tsx

```diff
@@ -106,16 +107,8 @@ export const keyTakeawaysElement = createReactAltStylesElementSpec(
   (fields) => fields.repeater,
   (repeaterChild) => (
     <>
-      <FieldWrapper
-        headingLabel="Key Takeaway Title"
-        field={repeaterChild.title}
-        useAlternateStyles={true}
-      />
-      <FieldWrapper
-        headingLabel="Key Takeaway Content"
-        field={repeaterChild.content}
-        useAlternateStyles={true}
-      />
+      <AltFieldWrapper field={repeaterChild.title} />
+      <AltFieldWrapper field={repeaterChild.content} />
```

Comment:
does this meet rule of 3, and should the abstraction not live in this file rather than FieldWrapper.tsx

Reply:
I don't see how it meets the rule of three, can you clarify? Happy to move the logic around

Reply:
I meant, does it need an abstraction if there are only two?